platform_versioned_docs/version-24.1/enterprise/_templates/k8s/data_studios/proxy.yml

```diff
@@ -19,13 +19,13 @@ spec:
         - name: proxy
           image: cr.seqera.io/private/nf-tower-enterprise/data-studio/connect-proxy:0.7.2
           command:
-            - caddy
             - serve
+            - caddy
```

Comment:
issue: this change should be reverted, as the order of the command in the list matters, and it should be `caddy server`, not `serve caddy`. and the same applies below